Framework/API/inc/MantidAPI/HistoryView.h

```diff
@@ -65,6 +65,12 @@ class MANTID_API_DLL HistoryView {
   const std::vector<HistoryItem> &getAlgorithmsList() const {
     return m_historyItems;
   };
+
+  std::shared_ptr<std::vector<Mantid::API::HistoryItem>>
+  getMutableAlgorithmsList() {
```

Comment:
Is this function used anywhere?

Reply:
Nope, it was left over from a previous fix that was later re-wrote. Removed.